config.yaml

```diff
@@ -26,6 +26,41 @@ log:
     ## to set whether or not the SDK key is included in the logging output.
     includeSdkKey: true
 
+##
+## tracing: tracing configuration
+##
+tracing:
+    ## bydefault tracing is disabled
+    ## to enable tracing set enabled to true
+    enabled: false
+    ## tracing exporter configuration
+    exporter:
+        ## bydefault stdout exporter is enabled
+        ## to enable remote exporter set default as "remote"
+        default: "stdout"
+        ## tracing service name
+        serviceName: "optimizely-agent"
+        ## tracing environment name
+        ## example: for production environment env can be set as "prod"
+        env: "dev"
+        ## tracing service configuration
+        services:
+            ## stdout exporter configuration
+            stdout:
+                ## for stdout tracing data is saved in the specified file
+                filename: "trace.out"
+            ## remote exporter configuration
+            remote:
+                ## remote collector endpoint
+                endpoint: "localhost:4317"
```

Comment:
Yes, this can be connected with any trace collector. I have tested with Jaeger locally. I can make sure the same config will work for datadog as well.

As this is working as push-model (agent will push trace data). Here remote means a remote trace collector which can be any vendor that's why skipping any vendor name.

Reply:
Authorization token for vendors?

Reply:
Yeah, I also looked for it inside OpenTelemetry. But surprising don't see any such functionality to pass such token so far. I think, agent (any tracing enabled server) & trace collector (example: Jaeger) will run inside private network and might not need any such token or security. But collector UI i.e. to see/explore those data can need token. that will depend on that vendor settings.

Reply:
I have found these available options [here](https://github.com/open-telemetry/opentelemetry-go/blob/main/exporters/otlp/otlptrace/otlptracehttp/options.go#L63-L116)